README.md

```diff
@@ -0,0 +1,713 @@
+var data = diacritics.getDecompose(["ss"]);
+/* data =>
+{
+    "de": {
+        "metadata": {...},
+        "data: {
+            "ß": {...}
+        }
+    }
+}
+*/
+```
+
+If no matching decomposes are found in the database, an object with an error "message" is returned.
+
+```js
+var diacritics = require("diacritics-transliterator");
+var test = diacritics.getDecompose(["&", "test"]);
+// test => { "message": "No matching decomposes found" }
+```
+
+## Transliteration functions
+
+### diacritics.transliterate(string [, type][, variant])
+
+Replaces diacritics within the string with either the base or decomposed value.
+
+- The `string` parameter (type: `string`) will replace any diacritic characters with base or decomposed values. Any non-diacritics characters will not be modified.
+- The `type` parameter (type: `string`) is set to either "base" or "decompose". Defaults to "base". If a "base" value is not found, a "decompose" value will be used instead; and vice-versa.
+- The `variant` optional parameter (type: `string`) is set using the language variant. If no variant is provided, the first matching language for the diacritic will be used.
+
+The returned value will be a string with all matching diacritics replaced with the set "base" or "decompose" value.
+
+**Note** that if a `variant` parameter is set and the string contains a diacritic not found in that language variant, it will not be processed!
+
+```js
+var transliterate = require("diacritics-transliterator").transliterate;
+var german = transliterate("¿abcñ-ß123?", "decompose" "de"); // "¿abcñ-ss123?"
+var spanish = transliterate("¿abcñ-ß123?", "base", "es"); // "?abcn-ß123?"
+// The base of ß is still ß
+var generic = transliterate("¿abcñ-ß123?"); // "?abcn-ß123?"
+// If no diacritics match the set language variant, the original string is returned
+var unchanged = transliterate("¿abcñ-ß123?", "base", "test"); // "¿abcñ-ß123?"
+```
+
+If no matching diacritics are found in the string or database, the original string is returned.
+
+### diacritics.createRegExp(string [, options])
+
+This function creates a regular expression that matches, or ignores, diacritics in the given `string`.
+
+The regular expression can:
+
+- Match all diacritics with or without their equivalents.
+- Match only non-diacritics, which ignores the diacritic and allows matching any character(s).
+- Match all diacritics in a case-insensitive manner.
+- Ignore any joiners (soft hyphens, zero width space, zero width non-joiner and zero width joiners) that may occur within the string.
+- Match the original string if no diacritics are found.
+
+The function uses the following parameters:
+
+- The `string` parameter (type: `string`) contains text with or without diacritic characters to be processed into a regular expression that matches this value.
+- The `options` parameter (type: `object`) contains the following settings:
+
+<table>
+  <thead>
+    <tr>
+      <th>Option
```

Comment:
Closing tags are optional... this makes it easier to read

ref: https://www.w3.org/TR/html5/syntax.html#optional-tags